core/src/main/java/com/datastrato/gravitino/connector/BaseCatalog.java

```diff
@@ -65,6 +68,18 @@ public abstract class BaseCatalog<T extends BaseCatalog>
   @Evolving
   protected abstract CatalogOperations newOps(Map<String, String> config);
 
+  /**
+   * Create a new instance of HasCapabilities, if the child class has special capabilities, it
+   * should implement this method to provide a specific HasCapabilities instance regarding that
+   * catalog.
+   *
+   * @return A new instance of HasCapabilities.
+   */
+  @Evolving
+  protected HasCapabilities newCaps() {
```

Comment:
I don't suggest use the abbreviation "caps" here, it is not a common abbreviation.

Reply:
renamed